### DISCUSSION TITLE
Copilot X access for Master's thesis project

### DISCUSSION THREAD

**Discussion by tharropoulos** [community/community#60692] [Copilot Conversations] [answered]
### Hi everyone,

I'm a senior student at the [department of Electrical and Computer Engineering of the Aristotle University of Thessaloniki](https://ee.auth.gr/en/) . As part of my studies, I have to conduct a personal thesis project, in collaboration with a professor of the department. We thought that with the prevalence of AI assistant tools, using a tool like copilot for creating software engineering projects would be a  perfect fit for a thesis. We've decided upon using copilot for this project, but the use of chat for better communication of ideas with the assistant, and using it for writing the tests of our features has become mandatory. I have been in the wait list for about a month now, and sadly I have to ask if I could have a bit of early access in order to proceed with my research. You can contact me directly at ftharropoulos@gmail.com for more information regarding my research.

**Comment by davecheney** [comment, marked as answer]
>  I have been in the wait list for about a month now, and sadly I have to ask if I could have a bit of early access in order to proceed with my research. 

I cannot find any record of your account signing up to the copilot chat waitlist. please apply again and I think you'll be invited quickly.

**Reply by tharropoulos** [reply]
![image](https://github.com/community/community/assets/93339170/eaf0cb90-c32a-4dd6-a714-d011793d6c32)
This is the docs waitlist

And this is the chat watilist
![image](https://github.com/community/community/assets/93339170/4d44f9a7-6afe-41ef-9303-941572606f91)

**Reply by davecheney** [reply]
My apologies. Normally when people say copilot x, they mean chat. I’m afraid I’m not a part of the team running the docs trial.

**Reply by tharropoulos** [reply]
No worries, I hope the chat feature will be available soon enough. I joined the waitlist now 😊
